test/Transactions/Orleans.Transactions.Tests/TransactionTestConstants.cs

```diff
@@ -14,5 +14,7 @@ public static class TransactionTestConstants
         // grain implementations
         public const string TransactionOrchestrationGrain = "Orleans.Transactions.Tests.TransactionOrchestrationGrain";
         public const string SingleStateTransactionalGrain = "Orleans.Transactions.Tests.SingleStateTransactionalGrain";
+        public const string DoubleStateTransactionalGrain = "Orleans.Transactions.Tests.DoubleStateTransactionalGrain";
```

Comment:
I understand this change is consistent with the previous pattern, but that pattern is error prone.  Suggest static readonly strings which get their value from the type, rather than constants.  The reason for this is that using the types provides compile time validation to the value.

Reply:
we cannot do it because these variable are used as `InlineData` which requires them to be const which I think might be why they are const in the first place

Reply:
Ah, fair point.. sux, but don't see good way around.. nm